### DISCUSSION TITLE
코드블록 copy해주는 모듈 추가

### DISCUSSION THREAD

**Discussion by ghdic** [pronist/hELLO#160] [Ideas]
https://github.com/arronhunt/highlightjs-copy

코드블록 copy 모듈을 추가할 수 있도록 옵션을 만들어주세요!

![image](https://user-images.githubusercontent.com/28496724/207018454-5f87160d-ab5c-45a8-83b3-f2f4b87a576d.png)

**Comment by TetraTheta** [comment]
사견입니다만, 코드 블록 복사가 필요한 경우, 사용자가 직접 스킨 편집을 통해 다음 문구를 추가하는 게 나을 것 같습니다.
```html
<head>
  <script src="https://unpkg.com/highlightjs-copy/dist/highlightjs-copy.min.js"></script>
  <link rel="stylesheet" href="https://unpkg.com/highlightjs-copy/dist/highlightjs-copy.min.css">
  <script>
  hljs.addPlugin(new CopyButtonPlugin());
  </script>
</head>
```

코드 블록 복사 버튼은 호불호가 갈릴 수 있는 영역이라고 생각합니다. 어떤 사람은 읽는 이로 하여금 편하게 코드를 복사해갈 수 있도록 버튼을 제공하는 것을 선호할 것이고, 다른 사람은 그러한 버튼 자체를 다른 이유로 인해 선호하지 않을 수 있습니다.

링크된 레포지토리를 간략하게나마 확인해보았는데, 적용에 있어 큰 어려움이 있어 보이지는 않아 보입니다. 만약 이를 사용자가 적용했을 때 레이아웃 문제가 발생한다면 그에 대해 추가로 논의할 가치가 있겠지만, 그게 아니라면 이 부분은 개인의 선택에 맡기는 것이 제일 좋아 보입니다.

**Comment by pronist** [comment]
@TetraTheta 사실 추가할 까 싶었지만, 저 레포지토리에 쓰인 코드가 너무 구립니다. 리팩토링하기 귀찮아서 안 넣었다는 것이 포인트.
